policy/release/external_parameters.rego

```diff
@@ -0,0 +1,56 @@
+#
+# METADATA
+# title: external parameters
+# description: >-
+#   Verify the attribute .predicate.buildDefinition.externalParameters of a
+#   SLSA Provenance v1.0 matches the expectation.
+#
+package policy.release.external_parameters
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+
+# METADATA
+# title: pipeline run params
+# description: >-
+#   Verify the PipelineRun was initialized with a set of expected parameters.
+#   By default it asserts git-repo, git-revision, and otuput-image are provided
+#   with non-empty values. This is configurable by the rule data key
+#   "pipeline_run_params". Any additional parameters are NOT allowed.
+# custom:
+#   short_name: pipeline_run_params
+#   failure_msg: PipelineRun params, %v, do not match expectation, %v.
+#
+deny contains result if {
+	some provenance in lib.pipelinerun_slsa_provenance_v1
+	param_names := {name |
+		some p in provenance.predicate.buildDefinition.externalParameters.runSpec.params
```

Comment:
Should getting the params be moved to a function like `buildParameters` or something similar in `attestations.rego`?